### ISSUE TITLE
Why set self.noobj_scale=100 ?

### ISSUE THREAD

**Issue by pxq0312** [eriklindernoren/PyTorch-YOLOv3#309]
During training, negative samples should be much more than positive samples. Is it supposed to set  a large weight for self.obj_scale instead of self.noobj_scale?

**Comment by zeruniverse** [comment]
Because you averaged the loss over number of boxes. Since negative samples are much more, the error of one false positive will be much smaller due to the large denominator.

**Comment by Flova** [comment]
This part of the code has been updated or removed. I, therefore, close this issue now. Feel free to reopen for further discussion.
